numpyro/hmc_util.py

```diff
@@ -711,11 +711,11 @@ def euclidean_kinetic_energy(inverse_mass_matrix, r):
     return 0.5 * np.dot(v, r)
 
 
-def potential_energy(model, model_args, model_kwargs, inv_transforms, skip_dist_transforms=True):
+def potential_energy(model, model_args, model_kwargs, inv_transforms):
```

Comment:
I think, at some point, we should document the requirements on params - namely they should be unconstrained, and be from the base distribution.